pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -11,6 +11,12 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/client-go/util/retry"
+
+	"github.com/rancher/rancher/pkg/features"
+
+	"golang.org/x/mod/semver"
+
 	"github.com/pkg/errors"
```

Comment:
Generally in rancher code the blocks are separated into two blocks: go libraries and other. Sometimes rancher developers divide the latter into multiple blocks, one for each owner but I usually advise against that since it's not a widespread practice here. I don't think either is the case here. Please update the import blocks of this code to be consistent. Another example is in the import scope containing [this](https://github.com/rancher/rancher/pull/40104/files#diff-f98141dc0b0f4c98b013c7362f5da9405d54be68694890c35caeff90609d2fd0R6) the block is a separate line for a rancher package but the latter block also contains a rancher package.

Reply:
This was likely the work of `goimports -w`, is that not the recommendation?

I saw CI failing because of the lack of `goimports` and assumed it was standard to run it over the files.

Reply:
Yeah but it'll just sort them by the block it finds them in AFAIK. You can remove the new lines and run it again.

Reply:
I expected `goimports` to do the empty line cleanup, but it doesn't do 😕